en/orm.rst

```diff
@@ -25,56 +25,55 @@ database connections <database-configuration>`.
 Quick Example
 =============
 
-To get started you don't have to write any code. If you've followed the :ref:`CakePHP
-conventions for your database tables <model-and-database-conventions>`
-you can just start using the ORM. For example if we wanted to load some data 
-from our ``articles`` table we could do::
+To get started you don't have to write any code. If you've followed the
+:ref:`CakePHP conventions for your database tables
+<model-and-database-conventions>` you can just start using the ORM. For example
+if we wanted to load some data from our ``articles`` table we would start off
+creating our ``Articles`` table class. Add the following to
+**src/Model/Table/ArticlesTable.php** after the ``<?php`` opening tag::
 
-    use Cake\ORM\Locator\LocatorAwareTrait;
+    <?php
+    namespace App\Model\Table;
 
-    public function someMethod()
+    use Cake\ORM\Table;
+
+    class ArticlesTable extends Table
     {
-        $articles = $this->getTableLocator()->get('Articles');
+    }
+
+Then in a controller, or command we can have CakePHP create an instnace for us::
 
-        $resultset = $articles->find()->all();
+    public function someMethod()
+    {
+        $this->loadModel('Articles');
+        $resultset = $this->Articles->find()->all();
 
         foreach ($resultset as $row) {
             echo $row->title;
         }
     }
-    
-Within a static method you can use the :php:class:`~Cake\\Datasource\\FactoryLocator` 
-to get the table locator::
-
-    $articles = FactoryLocator::get('Table')->get('Articles');
-
-Note that we didn't have to create any code or wire any configuration up.
-The conventions in CakePHP allow us to skip some boilerplate code and allow the
-framework to insert base classes when your application has not created
-a concrete class. If we wanted to customize our ArticlesTable class adding some
-associations or defining some additional methods we would add the following to
-**src/Model/Table/ArticlesTable.php** after the ``<?php`` opening tag::
 
-    namespace App\Model\Table;
+In other contexts, you can use the ``LocatorAwareTrait`` to add methods that
+enable you to get ORM tables::
 
-    use Cake\ORM\Table;
+    use Cake\ORM\Locator\LocatorAwareTrait;
 
-    class ArticlesTable extends Table
+    public function someMethod()
     {
+        $articles = $this->getTableLocator()->get('Articles');
+        // more code.
     }
 
-Table classes use the CamelCased version of the table name with the ``Table``
-suffix as the class name. Once your class has been created you get a reference
-to it using the :php:class:`~Cake\\ORM\\Locator\\TableLocator` as before::
-
-    use Cake\ORM\Locator\LocatorAwareTrait;
+Within a static method you can use the :php:class:`~Cake\\Datasource\\FactoryLocator` 
+to get the table locator::
 
-    $articles = $this->getTableLocator()->get('Articles');
+    $articles = FactoryLocator::get('Table')->get('Articles');
```

Comment:
This should be using TableRegistry::getTableLocator()-> instead IMO.
The magic string one is a bit weird for app level usage.

Reply:
This example has been in the docs for a while.

Reply:
Yeah, i still think its wrong to leak that internal detail.